components/sections/Footer/index.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import Image from 'next/image';
+import type { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import NavItem from '@/components/sections/NavItem';
+import { useSiteConfig } from '@/hooks/useSiteConfig';
+
+import styles from './index.module.css';
+
+const Footer: FC = () => {
+  const { footerLinks, socialLinks } = useSiteConfig();
+
+  return (
+    <div className={styles.footer}>
```

Comment:
We can use the [footer](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-footer-element.html) element there